srcpkgs/nautilus/template

```diff
@@ -18,10 +18,9 @@ short_desc="GNOME file manager"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Nautilus"
-#changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/master/NEWS"
-changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/gnome-45/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/main/NEWS"
```

Comment:
Comment this and add `changelog="https://gitlab.gnome.org/GNOME/nautilus/-/raw/gnome-46/NEWS"` below.